alpine/configs/collectd/filtering_config/filtering.conf

```diff
@@ -0,0 +1,58 @@
+LoadPlugin match_regex
+
+<Chain "PostCache">
+  <Rule> 
+    <Match regex>
+      Plugin "^cpu$"
+    </Match>
+    <Target write>
+      Plugin "aggregation"
+    </Target>
```

Comment:
Will need to address this after some additional validation.  I suspect that if the user were to totally disable the metadata plugin, this would spam the user with log messages.  Will likely need to make this an toggle-able setting from run.sh